### PR TITLE
Fix Point, Dimension and Bounds types

### PR DESCRIPTION
**Pull request by vespakoen**
...

Btw, just looking through your code and finding this: https://github.com/pravdomil/pravdomil-scripts-for-indesign/tree/master/js/html_export/lib#L48

Evil genius haha, hats off to you.

### CODE REVIEW

File: shared/ScriptUI.d.ts [hunk 1]
```diff
@@ -2373,22 +2398,22 @@ declare class UIEvent {
  */
 declare class Event {
 	/**
-	 *
```
Comment (pravdomil) [inline shared/ScriptUI.d.ts]: please leave the spaces there

File: shared/ScriptUI.d.ts [hunk 2]
```diff
@@ -2162,113 +2162,138 @@ declare class Panel extends _Control {
...
+
+  /**
+   * The vertical coordinate, a pixel offset from the origin of the element's coordinate space.
+   */
+  y: number;
+
+}
+
+declare type ArrayPoint = [number, number]
+
```
Comment (pravdomil) [inline shared/ScriptUI.d.ts]: you have used the double space as indent?

**Comment by pravdomil** [comment]
yea, that really does't work, here is typescript source
https://github.com/pravdomil/pravdomil-scripts-for-indesign/blob/072f08a770ce85c4c151050ef8b554270ffec8b3/html_export/lib/settings.ts#L52

**Comment by vespakoen** [comment]
Ahh, damn autofixer messed up the indentation =)

Also, you should be able to get rid of all the casts now (`as RadioButton`), I guess we just add ` | undefined` to the Bounds type assuming that's the error you are having?

**Comment by pravdomil** [comment]
the type assertion `as RadioButton` solves the CheckBox class collision issues

File: shared/ScriptUI.d.ts (hunk 1 above)
Comment (vespakoen) [reply shared/ScriptUI.d.ts]: Why if I may ask?

**Comment by pravdomil** [comment]
check https://github.com/pravdomil/types-for-adobe/pull/23/files

and let me know if its ready to merge

**Comment by vespakoen** [comment]
LGTM!

**Pull request merged**